### ISSUE TITLE
Redesign a Filter pages

### ISSUE THREAD

**Issue by Shalinimariappan** [codervivek5/VigyBag#1944]
A filter is not look good in UI. So I want to change a design in al pages which have filters. Please assign this to me , I want to contribute in this one.

**Comment by codervivek5** [comment]
kindly share some design if you find something

**Comment by codervivek5** [comment]
kindly connect in discord for more communication and share design with your issue number

**Comment by Shalinimariappan** [comment]
Yeah, Hi.. There have many issues are raised in filter UI because It fixed in left of a page
1. It was not comfortable for users
2. Responsive is not workable when It is in left side
3. UI is not designed for a every product page

Here is a solution!!

![Desktop - 1 (2)](https://github.com/user-attachments/assets/434de9d6-d12b-49b6-923a-5fac7631ffb2)

This UI contain vertical filter options,

1. Looks clean and clear
2. It makes also responsiveness easily

**Comment by Shalinimariappan** [comment]
Pls review it. after you said, I will submit a design to a Figma design file and I'm ready to work for a responsive too
